### PR TITLE
healthcare API: FHIR v1 formatting fixes and cleanup

### PR DESCRIPTION
**Pull request by noerog**
## Description

This is a continuation of https://github.com/GoogleCloudPlatform/python-docs-samples/pull/5691 but for v1 FHIR.

Continuing fixes for:

b/183816408
b/158653137
b/159065286

## Checklist
- [X] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [X] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [X] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [X] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [X] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Review by dandhlee — APPROVED** [review]
LGTM.

File: healthcare/api-client/v1/fhir/README.rst [hunk 1]
```diff
@@ -100,22 +100,49 @@ To run this sample:
     positional arguments:
       {create-fhir-store,delete-fhir-store,get-fhir-store,get-fhir-store-metadata,list-fhir-stores,patch-fhir-store,import-fhir-resources,export-fhir-store-gcs,get_iam_policy,set_iam_policy}
         create-fhir-store   Creates a new FHIR store within the parent dataset.
-        delete-fhir-store   Deletes the specified FHIR store.
-        get-fhir-store      Gets the specified FHIR store.
+                            See https://github.com/GoogleCloudPlatform/python-
+                            docs-samples/tree/master/healthcare/api-client/v1/fhir
+                            before running the sample.
```
Comment (tmatsuo) [inline healthcare/api-client/v1/fhir/README.rst:105]: Re: `See https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/healthcare/api-client/v1/fhir before running the sample.`

Does it make sense to add this sentence at the beginning of the README (or anywhere else you think appropriate)? It bulks up the documentation a lot and lost the brevity.

Not blocking though.

**Review by tmatsuo — APPROVED** [review]

File: healthcare/api-client/v1/fhir/README.rst (hunk 1 above)
Comment (noerog) [reply healthcare/api-client/v1/fhir/README.rst:105]: I did this following the "Descriptive to-level comment" in our rubric. Do you think it's best to just have it once at the top of the Python file? I wanted it to be visible to people copy/pasting snippets from the docs.

Comment (tmatsuo) [reply healthcare/api-client/v1/fhir/README.rst:105]: Ok, it sounds good if you have a reason.

Comment (noerog) [reply healthcare/api-client/v1/fhir/README.rst:105]: Actually looking at it now it does make sense to manually remove that sentence from each method in README.rst. We'll just have to do the manual removal any time we auto-gen README.rst.

**Pull request merged**